lang/cs/TestRunner.proj

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'd rather keep this project as dependency-free as possible. We want to be able to run tests without rebuilding.